pkg/capabilities/oci/manifest/types.go

```diff
@@ -8,30 +8,30 @@ import (
 )
 
 type OciImageManifestResponse struct {
-	image *specs.Manifest
-	index *specs.Index
+	Image *specs.Manifest `json:"image"`
```

Comment:
Exporting these fields is needed because the json lib cannot see them using reflect unless they are in the same package. A workaround would be to create a new helper type, analogous to this one, with the exported fields, but at this point, why not change it here.